### PR TITLE
Ignore checksum at end of track 2

### PR DESCRIPTION
**Pull request by abecevello**
Updates the track parser to ignore the checksum at the end of track 2.

@ryanbalsdon @davidseal

### CODE REVIEW

**Comment by ryanbalsdon** [comment]
lgtm

File: lib/magnet/parser.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ module Magnet
   class Parser
     TRACKS = {
       1 => /\A%(?<format>[A-Z])(?<pan>[0-9 ]{1,19})\^(?<name>[^^]*)\^\s?(?<expiration>\d{4}|\^)(?<service_code>\d{3}|\^)(?<discretionary_data>[^\?]*)\?\Z/,
```
Comment (ryanbalsdon) [inline lib/magnet/parser.rb]: Why not add it to track 1 too.

Comment (abecevello) [inline lib/magnet/parser.rb]: I don't have a test case that has a value in track 1. Should I just put it in as a similar character that was used in Track 2?

Comment (ryanbalsdon) [inline lib/magnet/parser.rb]: If it won't be used, I wouldn't bother. We can add it when we need the feature and have a real test case.

**Pull request merged**